components/app/project-detail/ProjectSingleDaoProposal.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div>
+    <h4>DAO <span class="text-primary">proposal</span></h4>
+
+    <div class="flex flex-col justify-between md:flex-row mt-8">
+      <!-- Start Date -->
+      <div class="flex justify-between my-1 md:flex-col">
+        <div class="flex">
+          <img
+            class="mr-1"
+            src="@/assets/images/icons/start-fund.svg"
+            alt="funding-start"
+          />
+
+          <p class="text-primary">Start</p>
+        </div>
+
+        <p class="small-text" v-text="start" />
```

Comment:
question (blocking): we should make sure we are consistent with our code style and either use v-text or the mustache syntax {{ start }}. If you are using v-text everywhere then leave is otherwise please update to mustache syntax.